client/reader/stream/empty.jsx

```diff
@@ -24,15 +23,15 @@ class FollowingEmptyContent extends React.Component {
 	};
 
 	render() {
-		const action = isDiscoverEnabled() ? (
```

Comment:
Quick fix: I noticed that this button shouldn't be gated on Discover being enabled. It must have pointed there originally but now sends the user to `/read/search`.

Reply:
oooh, so was this button broken for anyone non-english speakers?